lifelong_mapping/TopicInfo    info
lifelong_mapping/RawMessage[] messages
